@xen-orchestra/fs/src/local.js

```diff
@@ -1,10 +1,14 @@
 import df from '@sindresorhus/df'
 import fs from 'fs-extra'
-import { fromEvent } from 'promise-toolbox'
+import { fromEvent, retry } from 'promise-toolbox'
 
 import RemoteHandlerAbstract from './abstract'
 
 export default class LocalHandler extends RemoteHandlerAbstract {
+  constructor(remote, { tries = 10 } = {}) {
+    super(remote)
+    this.tries = tries
```

Comment:
Private property, prefix with `_`:

```suggestion
    this._tries = tries
```